schemas/configuration/v4/unattended_installation.json.in

```diff
@@ -4,9 +4,6 @@
     "type": "object",
     "properties": {
         "admin_password": {"type": "string"},
-        "host_access_control_config": {
-            "$ref": "host_access_control_config.json"
-        },
```

Comment:
Review changes to this file.

Reply:
Yeah, I'm going to remove this commit.

This should be handled by a separate PR.

Reply:
done